treeherder/webapp/api/jobs.py

```diff
@@ -91,10 +94,27 @@ def list(self, request, project, jm):
         - return_type (dict)
         """
         MAX_JOBS_COUNT = 2000
+        LAST_MODIFIED_WINDOW = 30  # minutes
+
         filter = UrlQueryFilter(request.query_params)
 
         offset = int(filter.pop("offset", 0))
         count = int(filter.pop("count", 10))
+        if "last_modified" in filter.conditions:
+            datestr = filter.get("last_modified")[1]
+            try:
+                last_modified = parser.parse(datestr)
+                if last_modified < datetime.datetime.utcnow() - \
+                   datetime.timedelta(minutes=LAST_MODIFIED_WINDOW):
```

Comment:
This constant is being re-computed for every web request, perhaps it should be broken out at some point?